tests/serverpod_test_server/test_integration/database_operations/default/integer/int_default_persist_test.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_test_server/src/generated/protocol.dart';
+import 'package:serverpod_test_server/test_util/test_serverpod.dart';
+import 'package:test/test.dart';
+
+void main() async {
+  var session = await IntegrationTestServer().session();
+
+  group('Given a class with "defaultPersist" fields,', () {
+    tearDownAll(() async => IntDefaultPersist.db.deleteWhere(
+          session,
+          where: (_) => Constant.bool(true),
+        ));
+
+    test(
+      'when creating a record in the database, then the "defaultPersist=10" field should be null',
+      () async {
+        var object = IntDefaultPersist();
+        var databaseObject = await IntDefaultPersist.db.insertRow(
+          session,
+          object,
+        );
+        expect(databaseObject.intDefaultPersist, isNull);
```

Comment:
Suggestion: This looks strange, shouldn't the expected value be 10 here?

Reply:
Also this is already fixed!